config/locales/en.yml

```diff
@@ -20,6 +27,14 @@ en:
               too_short: 'must include area code'
       messages:
         four_digit_year: 'year must be four digits'
```

Comment:
@threedaymonk is it correct like that? or should be something like
```ruby
models:
 concerns:
    person:
      messages:
         four_digit_year: 'year must be four digits'
```
I run the test like that and they passed. Then I delete those lines:
```ruby
       messages:
         four_digit_year: 'year must be four digits'
```
And run the tests again and they pass again. So I am not sure if we have to change it, as we added more details to locales.